src/playermat/PlaymatApi.ttslua

```diff
@@ -3,26 +3,70 @@ do
   local internal = { }
 
   local MAT_IDS = {
-    White = "8b081b",
+    White  = "8b081b",
     Orange = "bd0ff4",
-    Green = "383d8b",
-    Red = "0840d5"
+    Green  = "383d8b",
+    Red    = "0840d5"
   }
 
   local CLUE_COUNTER_GUIDS = {
-    White = "37be78",
+    White  = "37be78",
     Orange = "1769ed",
-    Green = "032300",
-    Red = "d86b7c"
+    Green  = "032300",
+    Red    = "d86b7c"
   }
 
   local CLUE_CLICKER_GUIDS = {
-    White = "db85d6",
+    White  = "db85d6",
     Orange = "3f22e5",
-    Green = "891403",
-    Red = "4111de"
+    Green  = "891403",
+    Red    = "4111de"
   }
 
+  -- Returns the by color requested playermat as object
+  ---@param matColor String Color of the playermat to return
+  PlaymatApi.getMatbyColor = function(matColor)
+    local mat = getObjectFromGUID(MAT_IDS[matColor])
+
+    if mat == nil then
+      broadcastToAll(playerColor .. " playermat could not be found!", "Yellow")
+    end
+    return mat
+  end
+
+  -- Returns the color of the by position requested playermat as string
+  ---@param startPos Table Position of the search, table get's roughly cut into 4 quarters to assign a playermat
+  PlaymatApi.getMatColorbyPosition = function(startPos)
+    if startPos.x < -42 then
+      if startPos.z > 0 then
+        return "White"
+      else
+        return "Orange"
+      end
+    else
+      if startPos.z > 0 then
+        return "Green"
+      else
+        return "Red"
+      end
+    end
+  end
+
+  -- Returns the draw deck of the requested playmat
+  ---@param matColor String Color of the playermat
+  PlaymatApi.getDrawDeck = function(matColor)
```

Comment:
Not worth blocking on here given our time constraint, but I'd remove this and add a "discard(count)" method to playmat, that's exposed from here.  That would make the playmat own its own components and handle operations on them, rather than passing them out for some unknown code to manipulate.

Reply:
Yeah, that will be useful once we have more scripting regarding discarding - let's defer this for now.